cub/cub/agent/agent_reduce.cuh

```diff
@@ -92,6 +92,48 @@ struct AgentReducePolicy : ScalingType
   static constexpr CacheLoadModifier LOAD_MODIFIER = _LOAD_MODIFIER;
 };
 
+/**
+ * Parameterizable tuning policy type for AgentReduce
+ * @tparam NOMINAL_BLOCK_THREADS Threads per thread block
+ * @tparam NOMINAL_WARP_THREADS_4B Threads per warp
+ * @tparam NOMINAL_ITEMS_PER_THREAD_4B Items per thread (per tile of input)
```

Comment:
The documentation, template parameter and member names are confusing. Do we mean nominal_4b_items here or just items? Like, are we talking about the actual `BLOCK_THREADS` for the value type used, or are we talking about the `BLOCK_THREADS` if the value type were the nominal_4B type (=`int`).

Also `NOMINAL_WARP_THREADS_4B` suggests it's the number of threads per warp if `int` was used, but it's going to recalculated based on the value type. But the documentation next to it says otherwise.

Please update the docs to match the intended semantics.

Reply:
We use same BLOCK_THREADS for both AgentReduce and AgentWarpReduce, but WARP_THREADS and ITEMS_PER_THREAD might differ based the reduce policy (i.e small or medium reduce policy) and are set based on 4B items. I have updated the docs and member names for BLOCK_THREADS, but others might remain as is, as they might be re-calculated based on MemBound Scaling.